### PR TITLE
Scan super classes for config items

### LINKED ISSUE #14626: quarkus.oidc.auth-server-url is missing in the all configuration guide
**Describe the bug**
`quarkus.oidc.auth-server-url` and other found in https://github.com/quarkusio/quarkus/blob/df5f7d27916db069d2bcc0744ced7e79d2b79584/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java are missing in the https://quarkus.io/guides/all-config#quarkus-oidc_quarkus-oidc-openid-connect section. 

**Expected behavior**

properties to be listed

**Actual behavior**
properties are not listed

**To Reproduce**

Visit https://quarkus.io/guides/all-config#quarkus-oidc_quarkus-oidc-openid-connect and observe missing properties. 

**Additional context**

I think the generator is not scanning config items from super classes, in this case https://github.com/quarkusio/quarkus/blob/df5f7d27916db069d2bcc0744ced7e79d2b79584/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java  hence that's why they are not showing. 

/cc @miguelsorianod thanks for spotting this.

### PR DESCRIPTION
**Pull request by machi1990**
Fixes https://github.com/quarkusio/quarkus/issues/14626

Verify that the generated `quarkus-oidc.adoc` file now contains `auth-server-url` and other config items defined in [OidcCommonConfig](https://github.com/quarkusio/quarkus/blob/df5f7d27916db069d2bcc0744ced7e79d2b79584/extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java)

/cc @sberyozkin for verification. 

/cc @miguelsorianod

### CODE REVIEW

**Comment by sberyozkin** [comment]
@machi1990 Hi, thanks very much, I'll verify a little bit later today, cheers

**Review by sberyozkin — APPROVED** [review]
@machi1990 - I've verified the generated `quarkus-oidc` and `quarkus-oidc-client` adocs now correctly include the inherited configuration

The build failure is not related
Thanks

File: extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
 
+@ConfigGroup
 public class OidcCommonConfig {
     /**
-     * The base URL of the OpenID Connect (OIDC) server, for example, 'https://host:port/auth'.
+     * The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
      * OIDC discovery endpoint will be called by default by appending a '.well-known/openid-configuration' path to this URL.
      * Note if you work with Keycloak OIDC server, make sure the base URL is in the following format:
-     * 'https://host:port/auth/realms/{realm}' where '{realm}' has to be replaced by the name of the Keycloak realm.
+     * `https://host:port/auth/realms/{realm}` where `{realm}` has to be replaced by the name of the Keycloak realm.
```
Comment (gsmet) [inline extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java:15]: Was it a good change given they are not formatted as Asciidoc by default?

Comment (machi1990) [reply extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java:15]: I am not sure I understood the concern, 

>  they are not formatted as Asciidoc by default?

Formatted by the generator?

**Pull request merged**